manager/testdata/unittests/write_asset.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: app.fybrik.io/v1alpha1
+kind: FybrikApplication
+metadata:
+  name: write-test
+  namespace: default
+  labels:
+    app: notebook
+spec:
+  selector:
+    clusterName: thegreendragon
+    workloadSelector:
+      matchLabels:
+        app: notebook
+  appInfo:
+    intent: Fraud Detection
+    role: Security
+  data:
+  - dataSetID: s3/redact-dataset
+    flow: write
+    requirements:
+      interface:
+        protocol: fybrik-arrow-flight
```

Comment:
protocol should be fybrik-flight (to differentiate from product flight which has different API), dataFormat should be arrow

Reply:
Do you mean that the module we use in the quick-start scenario should have a different definition?
What about e2e and notebook tests? 
@simanadler Please open a separate issue to handle the formats, taxonomy changes (otherwise all the tests will fail), module definitions. I don't think this should be done in this PR.

Reply:
@roee88 I thought you said that this was changed in the taxonomy.  Would appreciate it if you could weigh in here.